### ISSUE TITLE
Multi-language mode

### ISSUE THREAD

**Issue by damianodamiani** [chinacomx/chinacomx.github.io#19]
Add to new website the option to automatically translate into other languages, AI-enhanced?

**Comment by whalekeykeeper** [comment]
For the website user, if they are using Chrome, then they could just right-click their mouse and choose the "translation into xx" option. 

If we want to use some large language models to translate the website, doable, but the solution comes with cost and other issues.

**Comment by damianodamiani** [comment]
I agree, actually that is not something we are planning for anymore. 

However, maybe there is a (simple!) way to add an icon somewhere in the top menu bar that then links to the browser's translation function? to make it more prominent?

**Comment by whalekeykeeper** [comment]
I think for users who're relying a lot on translations they should already be familiar with the translation option built in Chrome. Therefore, not necessary.

**Comment by chinacomx-admin** [comment]
solved with #96
